controller/templates/server/api/template.controller.js

```diff
@@ -1,10 +1,8 @@
 (function IIFE() {
   'use strict';
 
-  var errorHandler = require('../../lib/errorHandler');
-
   function exampleMethod(req, res, next) {
-    return errorHandler.notFound(req, res);
+    return res.status(404).json({});
```

Comment:
I don't like this. Either send back a proper error or only `res.sendStatus(404);` 

http://jsonapi.org/format/#fetching-resources-responses-404